### ISSUE TITLE
Scrollbar for the Select component with more than 6 items on iPad not appearing until touch

### ISSUE THREAD

**Issue by zijianhuang** [angular/material2#8356]
#### Bug:

In iPad, the Material2 Select component with more than 6 options does not display a vertical scrollbar when dropping down, so the user would not know there are more options through a snapshot until touching the list. However, if the user touch-selects an option, the scrollbar will hardly be noticeable. This is bad for the affordance of the dropdown list.

#### What is the expected behavior?

In iPad, HTML Select+Option with many options will display a vertical scrollbar for around 1 second when dropping down, so the user would know there are more options through a snapshot, without touching the list.

In Android, the Material2 Select component with many options does display a vertical scrollbar for around 1 second when dropping down, so the user would know there are more options through a snapshot.

In iPad, the Select component should have similar behavior for affordance . 

#### What are the steps to reproduce?

In https://material.angular.io/components/select/overview, pick any example and add more options, you will see.

#### What is the use-case or motivation for changing an existing behavior?

1. Affordance for telling the user more options available. 
2. Being consist with the behavior on Android, and HTML Select+Option on iPad

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Angular 5, Material 5.0.0-rc0. All latest browsers. iPad 3 iOS 9.3; iPad Pro iOS 10.3.

**Comment by mmxcrono** [comment]
I agree, there are concerns for this in forms as the user is not aware of all options that are available unless they touch and see the scrollbar afterwards.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
